python/smqtk/utils/bit_utils.py

```diff
@@ -91,10 +91,35 @@ def neighbor_codes(b, c, d):
             yield c ^ fltr
 
 
+def _bit_vector_to_int_fast(v):
+    """
+    Transform a numpy vector representing a sequence of binary bits [0 | 1]
+    into an integer representation.
+
+    This version works only on contiguous arrays.
+
+    :param v: 1D Vector of bits
+    :type v: numpy.core.multiarray.ndarray
+
+    :return: Integer equivalent
+    :rtype: int
+    """
+    pad = '\x00' * (v.dtype.itemsize - 1)
+    if (
```

Comment:
Some odd white-spacing here.

Reply:
Roger that. This was standard on one of my recent projects, so you'll probably find a bunch more of these. I'll run through and change them all.